Source/WebCore/page/NavigationHistoryEntry.h

```diff
@@ -69,19 +71,27 @@ class NavigationHistoryEntry final : public RefCounted<NavigationHistoryEntry>,
         ReferrerPolicy referrerPolicy { ReferrerPolicy::Default };
     };
 
-    NavigationHistoryEntry(ScriptExecutionContext*, const DocumentState&, Ref<HistoryItem>&&, String urlString, WTF::UUID key, RefPtr<SerializedScriptValue>&& state = { }, WTF::UUID = WTF::UUID::createVersion4());
+    NavigationHistoryEntry(Navigation&, const DocumentState&, Ref<HistoryItem>&&, String urlString, WTF::UUID key, RefPtr<SerializedScriptValue>&& state = { }, WTF::UUID = WTF::UUID::createVersion4());
 
+    // ActiveDOMObject.
+    bool virtualHasPendingActivity() const;
```

Comment:
Is this missing an `override` or `final`?

Reply:
Yes it is! I'm surprised the compiler doesn't complain. I'll fix.